Services/ProjectService/Startup.cs

```diff
@@ -15,6 +15,12 @@
 using LT.DigitalOffice.ProjectService.Repositories.Interfaces;
 using LT.DigitalOffice.ProjectService.Validators;
 using MassTransit;
+using LT.DigitalOffice.Kernel.Broker;
+using Microsoft.AspNetCore.Builder;
```

Comment:
The using directives appeared previously in this namespace

Reply:
Fixed